### DISCUSSION TITLE
✨ Wallabag

### DISCUSSION THREAD

**Discussion by JigSawFr** [runtipi/runtipi-appstore#2195] [Requests]
### Description

wallabag is a web application allowing you to save web pages for later reading. Click, save and read it when you want. It extracts content so that you won't be distracted by pop-ups and cie.

You can install it on your own server, or you can create an account on [wallabag.it](https://wallabag.it/).

- Website: [wallabag.org](https://wallabag.org/)
- Android app: [wallabag/android-app](https://github.com/wallabag/android-app)
- iOS app: [wallabag/ios-app](https://github.com/wallabag/ios-app)
- Browser extension: [wallabag/wallabagger](https://github.com/wallabag/wallabagger)
- GNOME (Linux) app: [read-it-later](https://gitlab.gnome.org/World/read-it-later) (not maintained by this project)
- All resources about wallabag ecosystem are listed here: https://github.com/wallabag/wallabag/wiki/wallabag-ecosystem

### Official Website

https://wallabag.org/

### Documentation link

https://doc.wallabag.org/

### Last application release & date

2.6.8 (3/01/2024)

### Application license

MIT license

### Source code repository link

https://hub.docker.com/r/wallabag/wallabag/

### Docker image link

https://hub.docker.com/r/wallabag/wallabag/

### Please confirm the following

- [X] I confirm that I've put the app name in the title of the discussion
- [X] I confirm that the application is actively maintained.
- [X] I confirm that the docker image of the application is actively maintained.
- [X] I confirm that I have indeed checked that no similar request for this application already exists

### Gathering crowds

Hey folks!

Please upvote 👍 this discussion to show your interest in this request! 

Thanks ⛺

**Comment by JigSawFr** [comment]
Originally requested by @SpencerDub

**Comment by superuser-does** [comment]
This is an application which has no equivalent application in the store.

An alternative is Shaarli, which has its own [request page](https://github.com/runtipi/runtipi-appstore/discussions/2395).
